invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/JobRunsHeader.js

```diff
@@ -70,7 +71,12 @@ export class JobRunsHeaderComponent extends Component {
         </div>
         <div className="column ten wide right aligned">
           {loading ? null : (
-            <RunButton jobId={jobId} config={config} onError={this.onError} />
+            <RunButton
+              jobId={jobId}
+              config={config}
+              onError={this.onError}
+              queue={queue}
```

Comment:
```suggestion
              args={args}
              queue={queue}
              onError={this.onError}
```

1. nit: order of parameters :)
2. nit/other: not sure why it's `config` and not `args` so that it matches what is expected in the REST API payload